### ISSUE TITLE
Consider item options for selection widget when no mapping is defined

### ISSUE THREAD

**Issue by lolodomo** [openhab/openhab-android#912]
As discussed in https://github.com/eclipse/smarthome/issues/4942 it would be cool to implement this feature in the Android app too.
I just implemented it in Basic UI and Classic UI; it was already present in Paper UI.

**Comment by maniac103** [comment]
@lolodomo Is there a quick way to test this with the demo setup?

**Comment by mueller-ma** [comment]
There is an example sitemap in https://github.com/eclipse/smarthome/issues/4942#issue-289778778. I recommend to check the rest api if the values are present. If not, you need to update your oh server.

**Comment by maniac103** [comment]
Yes, but AFAICT an item which is linked to a special channel with options support is needed. My question is whether there's a way to have such a channel without the need for special hardware support.

**Comment by lolodomo** [comment]
No I don't think this is available in the demo sitemap. It requires using a binding that defines some options for a channel, either statically or dynamically.
I used the netatmo binding for my tests.

**Comment by mueller-ma** [comment]
Your PR worked for me. Maybe @lolodomo can test it as well so you don't have to test it.

**Comment by lolodomo** [comment]
Temperature trend and pressure trend.

**Comment by maniac103** [comment]
I don't see any trend channels either for the indoor or the outdoor sensor ... I guess this is some post-2.3 thing? If so, I'd like to rely on @mueller-ma 's testing :-)

**Comment by maniac103** [comment]
*facepalm* It appears when clicking 'Show more' ... stupid me. Thanks, I'll give that a try.
